src/app/SDLApp.cpp

```diff
@@ -66,7 +66,12 @@ SDLApp::SDLApp(const SDLApp::Settings &settings) :
     ImGui::CreateContext();
     ImGuiIO &imgui_io = ImGui::GetIO();
     ImGui_ImplSDL2_InitForOpenGL(_window, _glCtx);
-    ImGui_ImplOpenGL3_Init("#version 150");
+    ImGui_ImplOpenGL3_Init("#version 330");
+
+    ImGui::StyleColorsDark();
+    ImGuiStyle& style = ImGui::GetStyle();
+    style.WindowBorderSize = 0.f;
```

Comment:
This makes unfocused windows blend into the dark background, which makes it quite ugly and hard to tell where the to bar of the window ends.

Reply:
That is a good point